go-controller/pkg/libovsdb/ops/switch.go

```diff
@@ -312,6 +312,9 @@ func createOrUpdateLogicalSwitchPortsOps(nbClient libovsdbclient.Client, ops []l
 	m := newModelClient(nbClient)
 	ops, err := m.CreateOrUpdateOps(ops, opModels...)
 	sw.Ports = originalPorts
+	if err != nil && errors.Is(err, libovsdbclient.ErrNotFound) && !createSwitch {
+		err = fmt.Errorf("could not find switch: %q, %w", sw.Name, err)
+	}
```

Comment:
I was running into some bugs while running the tests and the addLogicalPort would fail and just say "object not found" i tracked down that it was because of the switch missing, so i made a quality of life improvement to make the error more descriptive

Reply:
you can see i have trust issues :)